content/docs/reference/api/aut/op-prot/_index.md

```diff
@@ -442,14 +416,6 @@ aut protocol set-minimum-base-fee [OPTIONS] base-fee
 {{< /tab >}}
 {{< /tabpane >}}
 
-<!--
```

Comment:
So we are sure to not use the nodejsconsole anymore?

Reply:
I've changed the PR name to make it easy to find should we want to restore `nodejsconsole` commands!